pkg/aspect/configError.go

```diff
@@ -77,5 +77,19 @@ func (e ConfigError) Error() string {
 }
 
 func (e *ConfigErrors) Error() string {
+	if e == nil || e.Multi == nil {
+		return ""
+	}
 	return e.Multi.Error()
 }
+
+// Extend joins 2 configErrors together.
+func (e *ConfigErrors) Extend(ee *ConfigErrors) *ConfigErrors {
```

Comment:
Here is an idea: instead of using string use proto reflection:
https://godoc.org/github.com/golang/protobuf/proto#StructProperties
I would drop multierror aspect from configerrors, and just let people use multierror directly.